web/src/app/icons/page.tsx

```diff
@@ -1,70 +1,83 @@
+	const description = getBrowseDescription(totalIcons)
+
 	return {
-		title: "Browse Icons | Free Dashboard Icons",
-		description: `Search and browse through our collection of ${totalIcons} curated icons for services, applications and tools, designed specifically for dashboards and app directories.`,
-		keywords: [
-			"browse icons",
-			"dashboard icons",
-			"icon search",
-			"service icons",
-			"application icons",
-			"tool icons",
-			"web dashboard",
-			"app directory",
-		],
+		title,
+		description,
+		keywords: BROWSE_KEYWORDS,
 		openGraph: {
-			title: "Browse Icons | Free Dashboard Icons",
-			description: `Search and browse through our collection of ${totalIcons} curated icons for services, applications and tools, designed specifically for dashboards and app directories.`,
+			title: `Browse Icons ${TITLE_SEPARATOR} ${SITE_NAME} ${TITLE_SEPARATOR} ${SITE_TAGLINE}`,
+			description,
 			type: "website",
-			url: `${BASE_URL}/icons`,
-			images: [
-				{
-					url: "/og-image.png",
-					width: 1200,
-					height: 630,
-					alt: "Browse Dashboard Icons Collection",
-					type: "image/png",
-				},
-			],
+			url: `${WEB_URL}/icons`,
+			images: [DEFAULT_OG_IMAGE],
 		},
 		twitter: {
 			card: "summary_large_image",
-			title: "Browse Icons | Free Dashboard Icons",
-			description: `Search and browse through our collection of ${totalIcons} curated icons for services, applications and tools, designed specifically for dashboards and app directories.`,
-			images: ["/og-image-browse.png"],
+			title: `Browse Icons ${TITLE_SEPARATOR} ${SITE_NAME} ${TITLE_SEPARATOR} ${SITE_TAGLINE}`,
+			description,
+			images: [DEFAULT_OG_IMAGE.url],
 		},
 		alternates: {
-			canonical: `${BASE_URL}/icons`,
+			canonical: `${WEB_URL}/icons`,
 		},
+		other: {
+			"revisit-after": "3 days",
+		}
 	}
 }
 
 export const dynamic = "force-static"
 
 export default async function IconsPage() {
 	const icons = await getIconsArray()
+
+	const gallerySchema = {
+		"@context": "https://schema.org",
+		"@type": "ImageGallery",
+		"name": `${SITE_NAME} - Browse ${icons.length} Icons - ${SITE_TAGLINE}`,
+		"description": getBrowseDescription(icons.length),
+		"url": `${WEB_URL}/icons`,
+		"numberOfItems": icons.length,
+		"creator": {
+			"@type": "Organization",
+			"name": ORGANIZATION_NAME,
+			"url": GITHUB_URL
+		}
+	}
+
 	return (
-		<div className="isolate overflow-hidden">
-			<div className="py-8">
-				<div className="space-y-4 mb-8 mx-auto max-w-7xl">
-					<div className="flex flex-col sm:flex-row sm:items-center justify-between gap-4">
-						<div>
-							<h1 className="text-3xl font-bold">Browse icons</h1>
-							<p className="text-muted-foreground">Search through our collection of {icons.length} beautiful icons.</p>
+		<>
+			<Script id="gallery-schema" type="application/ld+json">
+				{JSON.stringify(gallerySchema)}
+			</Script>
+			<Script id="org-schema" type="application/ld+json">
+				{JSON.stringify(ORGANIZATION_SCHEMA)}
+			</Script>
```

Comment:
Refactor to using the nextjs way https://nextjs.org/docs/app/building-your-application/optimizing/metadata#json-ld